vaadin-spreadsheet/src/main/java/com/vaadin/addon/spreadsheet/CellSelectionManager.java

```diff
@@ -281,13 +282,15 @@ private void handleCellAddressChange(int rowIndex, int colIndex,
      * This method is called when the cell selection has changed via the address
      * field.
      * 
-     * @param rowIndex
```

Comment:
I guess I did include a few code changes, to avoid assigning to method parameters.